packages/backend/src/models/SchedulerModel/index.ts

```diff
@@ -252,6 +254,20 @@ export class SchedulerModel {
         return this.getSchedulerAndTargets(schedulerUuid);
     }
 
+    async setSchedulerEnabled(
+        schedulerUuid: string,
+        enabled: boolean,
+    ): Promise<SchedulerAndTargets> {
+        await this.database(SchedulerTableName)
+            .update({
+                enabled,
+                updated_at: new Date(),
```

Comment:
Woah, I didn't know you had to set updated_at manually. Do you have to do this every time you update a database entry?

Reply:
in theory, it is not needed, because there is a trigger in db. However, this is kind of a type limitation in the model.